PublicSDKAPITester/RevenueCatAPI.m

```diff
@@ -0,0 +1,120 @@
+
+    //    [p presentCodeRedemptionSheet]; ///TODO: iOS ONLY, TEST FOR THIS API BY LOOKING UP SELECTOR
+    RCPurchases *p = [RCPurchases configureWithAPIKey:@""];
+    
+    [RCPurchases setLogHandler:^(RCLogLevel l, NSString *i) {}];
+    canI = [RCPurchases canMakePayments];
+    version = [RCPurchases frameworkVersion];
+    [RCPurchases addAttributionData:@{} fromNetwork:RCAttributionNetworkBranch];
+    [RCPurchases addAttributionData:@{} fromNetwork:RCAttributionNetworkBranch forNetworkUserId:@""];
+        
+    automaticAppleSearchAdsAttributionCollection = [RCPurchases automaticAppleSearchAdsAttributionCollection];
+    debugLogsEnabled = [RCPurchases debugLogsEnabled];
+    logLevel = [RCPurchases logLevel];
+    proxyURL = [RCPurchases proxyURL];
+    forceUniversalAppStore = [RCPurchases forceUniversalAppStore];
+    simulatesAskToBuyInSandbox = [RCPurchases simulatesAskToBuyInSandbox];
+    sharedPurchases = [RCPurchases sharedPurchases];
+    isConfigured = [RCPurchases isConfigured];
+    allowSharingAppStoreAccount = [p allowSharingAppStoreAccount];
+    finishTransactions = [p finishTransactions];
+    delegate = [p delegate];
+    appUserID = [p appUserID];
+    isAnonymous = [p isAnonymous];
+    
+    RCPurchaserInfo *pi = [[RCPurchaserInfo alloc] init];
+    SKProduct *skp = [[SKProduct alloc] init];
+    SKProductDiscount *skpd = [[SKProductDiscount alloc] init];
+    SKPaymentDiscount *skmd = [[SKPaymentDiscount alloc] init];
+    
+    RCPackage *pack = [[RCPackage alloc] init];
+    [RCPurchases configureWithAPIKey:@"" appUserID:@""];
+    [RCPurchases configureWithAPIKey:@"" appUserID:@"" observerMode:false];
+    [RCPurchases configureWithAPIKey:@"" appUserID:@"" observerMode:false userDefaults:nil];
+    
+    [p invalidatePurchaserInfoCache];
+
+    NSDictionary<NSString *, NSString *> *attributes = nil;
+    [p setAttributes: attributes];
+    [p setEmail: @""];
+    [p setPhoneNumber: @""];
+    [p setDisplayName: @""];
+    [p setPushToken: [@"" dataUsingEncoding: NSUTF8StringEncoding]];
+    [p setAdjustID: @""];
+    [p setAppsflyerID: @""];
+    [p setFBAnonymousID: @""];
+    [p setMparticleID: @""];
+    [p setOnesignalID: @""];
+    [p setMediaSource: @""];
+    [p setCampaign: @""];
+    [p setAdGroup: @""];
+    [p setAd: @""];
+    [p setKeyword: @""];
+    [p setCreative: @""];
+    [p collectDeviceIdentifiers];
+    
+    [p purchaserInfoWithCompletionBlock:^(RCPurchaserInfo *info, NSError *error) {}];
+    [p offeringsWithCompletionBlock:^(RCOfferings *info, NSError *error) {}];
+    [p productsWithIdentifiers:@[@""] completionBlock:^(NSArray<SKProduct *> *products) { }];
+    [p purchaseProduct:skp withCompletionBlock:^(SKPaymentTransaction *t, RCPurchaserInfo *i, NSError *error, BOOL userCancelled) { }];
+    [p purchasePackage:pack withCompletionBlock:^(SKPaymentTransaction *t, RCPurchaserInfo *i, NSError *e, BOOL userCancelled) { }];
```

Comment:
Does this compile correctly? IIRC #533 made the file `internal`, I had expected that calling methods that reference `RCPackage` would stop working until the migration was finalized

Reply:
It stopped working, but I have another PR with those changes incoming.